sonar-python-plugin/src/main/java/org/sonar/plugins/python/coverage/PythonCoverageSensor.java

```diff
@@ -95,19 +59,19 @@ public void execute(SensorContext context, Map<InputFile, Set<Integer>> linesOfC
     String baseDir = context.fileSystem().baseDir().getPath();
     Settings settings = context.settings();
 
+    LOG.info("Python unit test coverage");
```

Comment:
@pynicolas But if some message after is displayed (like report is not found) then it's not clear which report. We would have similar behaviour if it will be separate sensor.